pkg/tester/data.go

```diff
@@ -185,6 +186,10 @@ func InitializeData(
 		}
 	}
 
+	// If we enable all-in-memory or L0-in-memory mode, badger DB's TableSize and ValueLogFileSize will change
+	// according to users config. tableSize means the LSM table size, when the table more than the tableSize, 
```

Comment:
what does ValueLogFileSize do

Reply:
it is the maximum size of a single value log file
increasing it will increase the write speed